core/src/main/scala/org/apache/spark/sql/execution/columnar/impl/JDBCSourceAsColumnarStore.scala

```diff
@@ -254,9 +255,9 @@ class JDBCSourceAsColumnarStore(private var _connProperties: ConnectionPropertie
             // or it will be committed later
             val txId = SparkShellRDDHelper.snapshotTxIdForWrite.get
             if (txId != null && !txId.equals("null")) {
-              val statement = conn.createStatement()
-              statement.execute(
-                s"call sys.GET_SNAPSHOT_TXID('$txId')")
+              val statement = conn.prepareCall("call sys.GET_SNAPSHOT_TXID(?)")
+              statement.registerOutParameter(1, java.sql.Types.VARCHAR)
+              statement.execute()
```

Comment:
Did you see any issue? I am asking because the returned value was not used. Unless the procedure itself is not executed on server due to some exception?, there shouldn't have any issue.